apps/web/pages/_app.tsx

```diff
@@ -25,6 +32,15 @@ const calFont = localFont({
 });
 
 function MyApp(props: AppProps) {
+  useEffect(() => {
+    (window as any).intercomSettings = {
+      api_base: "https://api-iam.intercom.io",
+      name: "Jane Doe", // Full name – doesn't work yet
+      email: "customer@example.com", // Email address – doesn't work yet
+      created_at: 1312182000, // Signup date as a Unix timestamp – doesn't work yet
```

Comment:
this needs to be fixed and replaced with actual user data. couldn't find a way to make this work yet (not even with Jane Doe sample data)